README.md

```diff
@@ -1,250 +1,326 @@
-# Freifunk Berlin Firmware
-https://wiki.freifunk.net/Berlin:Firmware
+# Freifunk Berlin Firmware https://wiki.freifunk.net/Berlin:Firmware
+
+This is the build-system for the Firmware of Freifunk Berlin. The firmware is
+based on vanilla [OpenWrt](https://openwrt.org/start) with some modifications
+(to fix broken stuff in OpenWrt itself or for example LuCI) and additional
+default packages and configuration settings.
 
-This is the build-system for the Firmware of Freifunk Berlin.
-The firmware is based on vanilla [OpenWrt](https://openwrt.org/start) with some modifications (to fix
-broken stuff in OpenWrt itself or for example LuCI) and additional default packages/configuration settings.
 
 ## Contact / More information
 
-More user relevant information about the firmware are on the wiki page at: https://wiki.freifunk.net/Berlin:Firmware. There you can also find the
-* [ReleaseNotes](https://wiki.freifunk.net/Berlin:Firmware/v1.0.2)
-* a tutorial ([en](https://wiki.freifunk.net/Berlin:Firmware:En:Howto) / [de](https://wiki.freifunk.net/Berlin:Firmware:Howto)) on router configuration
+More user relevant information on the firmware are on the wiki page at:
+https://wiki.freifunk.net/Berlin:Firmware. There you can also find the 
+* [ReleaseNotes](https://wiki.freifunk.net/Berlin:Firmware/v1.0.2) and
```

Comment:
Hedy-1.0.6 is out and should be referenced here. But this can be a separate commit.

Reply:
Is it possible to use a variable which accesses the latest tagged release version.  (for the separate commit which will someday materialize)

Reply:
As the Readme belongs to the Codebase of a release, releasing something with an old or template value here, will end in having this template in the released Readme. Updating it after releasing creates a new version of the file (commit) which should cause  to release again. So the never ending loop has started ...

In long-term I suggest to get rid of the release-history in the Readme of the firmware-buildsystem, like I did here: https://github.com/freifunk-berlin/firmware/tree/745bda009b313cf3ce4eaacd82e232a4644f8a03

Reply:
As the page for v1.0.6 does not exist, I linked the page for v1.0.3